frontend/src/services/apiService.ts

```diff
@@ -21,9 +21,16 @@ import { jwt } from "./api/jwt";
 import { users } from "./api/users";
 import { fundings } from "./api/fundings";
 import { profile } from "./api/profile";
+import { courses } from "./api/courses";
 
 const baseApi = "http://localhost:8000/api/v1";
 
+export enum electionActiveStatus {
```

Comment:
Doesn't  belong  here

Reply:
Also capitalize `enum` names

Reply:
Check with master branch and removed this element